architecture/additional_concepts/storage.adoc

```diff
@@ -68,26 +68,26 @@ administrator configures one or more dynamic provisioners that provision storage
 and a matching PV.
 
 Alternatively, a cluster administrator can create a number of PVs in advance,
-which carry the details of the real storage that is available for use by cluster
-users. PVs exist in the API and are available for consumption.
+which carry the details of the real storage that is available for use. PVs exist
+in the API and are available for consumption.
 
 [[binding]]
 
 === Binding
 
-A user creates a `PersistentVolumeClaim` with a specific amount of storage
-requested and with certain access modes and optionally a `StorageClass`. A
-control loop in the master watches for new PVCs. It either finds a matching PV
-or waits for a provisioner for the `StorageClass` to create one, then binds them
-together.
+Create a PVC with a specific amount of storage requested and with certain access
+modes. Optionally, you can also create a storage class to describe and classify
+the storage. A control loop in the master watches for new PVCs. It either finds
+a matching PV or waits for a provisioner for the storage class to create one,
+then binds them together.
 
-The user will always get at least what they asked for, but the volume might be
-in excess of what was requested. This is especially true with manually
+You will always get the amount requested, but the volume might be
```

Comment:
maybe s/You will always get the amount requested, but the volume might be in excess of your request./The PV volume might exceed your requested volume.